sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -135,16 +135,6 @@ export interface ApplicationDataDto {
   student: ApplicationStudentDetails;
 }
 
-export interface StudentApplicationDTO {
```

Comment:
Instead of creating all the objects for this interface, we can reuse ActiveApplicationSummaryDTO and add award object to it additionally.

Reply:
Its already deleted guru and I am using `ActiveApplicationSummaryDTO`